bessctl/commands.py

```diff
@@ -1031,6 +1031,20 @@ def command_gatehook(cli, name, module, direction, gate, cmd, arg_type, args):
         cli.bess.resume_all()
 
 
+# Please do not rely on this API. This API may replaced with `command port PORT`
+# in the same way with gatehook and module
+@cmd('configure port PORT [PORT_ARGS...]', '[Experimental] Update a port configuration')
+def conigure_port(cli, name, args):
+    if args is None:
```

Comment:
style thing, feel free to ignore: `cli.bess.ser_port_config(name, args or {})` is shorter, without hurting readability (imo).